raiden/blockchain/filters.py

```diff
@@ -109,59 +110,55 @@ class StatelessFilter(LogFilter):
     Pass latest block_number to get_(new|all)_entries to avoid querying it
     """
 
-    def __init__(self, web3: Web3, filter_params: Dict[str, Any]) -> None:
+    def __init__(
+        self,
+        web3: Web3,
+        from_block: BlockNumber,
+        contract_address: ChecksumAddress,
+        topics: Optional[List[Optional[str]]],
```

Comment:
Making the filter fields explicit allows for proper type checking and simpler code, since we don't have to handle the possibility of a field being `None` anymore.